src/EditorFeatures/Test2/Simplification/ParenthesisSimplificationTests.vb

```diff
@@ -294,6 +294,84 @@ Class C
         Dim x = 1 &gt; (&lt;xml&gt;&lt;/xml&gt;)
     End Sub
 End Class
+</code>
+
+            Await TestAsync(input, expected)
+        End Function
+
+        <WorkItem(40442, "https://github.com/dotnet/roslyn/issues/40442")>
+        <Fact, Trait(Traits.Feature, Traits.Features.Simplification)>
+        Public Async Function TestVisualBasic_OnlyRemoveEmptyArgumentLisForMethodGroup() As Task
+            Dim input =
+<Workspace>
+    <Project Language="Visual Basic" CommonReferences="true">
+        <Document>
+Public Class TestClass
+    Shared Function GetFour() As Integer
+        Return 4
+    End Function
+    
+    Public Shared Sub Main()
+        Dim inferredMethodGroup = MyIf({|SimplifyExtension:TestClass.GetFour()|})
```

Comment:
please break into separate tests.  it will be much easier for diagnosing test failures and maintaining these.  thanks!